### PR TITLE
View, create, and use tokens with the API

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes # 1541

View, create, and use tokens with the API. Add command-line examples (`curl` + `jq`).

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/api-guide/api_intro.md [hunk 1]
```diff
@@ -31,11 +31,12 @@ an [InfluxDB Cloud account](/influxdb/cloud/sign-up).
 
 ### Authentication
 
-Before diving into the API, use the InfluxDB UI to
+InfluxDB uses [API tokens](/influxdb/v2.0/security/tokens/) to authorize API requests.
+
+1. Before diving into the API, use the InfluxDB UI to
```
Comment (pierwill) [inline content/influxdb/v2.0/api-guide/api_intro.md:36]: Less colloquial but still expressive? Just an idea.

```suggestion
1. Before exploring the API, use the InfluxDB UI to
```

File: content/influxdb/v2.0/api-guide/api_intro.md [hunk 2]
```diff
@@ -31,11 +31,12 @@ an [InfluxDB Cloud account](/influxdb/cloud/sign-up).
...
 [create an initial API token](/influxdb/v2.0/security/tokens/create-token/) for your application.
 
-InfluxDB uses [API tokens](/influxdb/v2.0/security/tokens/) to authorize API requests.
-Include your API token as an `Authorization` header in each request.
+2. Include your API token as an `Authorization: Token yourApiToken` header in each request.
```
Comment (pierwill) [inline content/influxdb/v2.0/api-guide/api_intro.md:39]:
```suggestion
2. Include your API token with each request as a head with the form: `Authorization: Token yourApiToken`.
```

File: shared/text/api/v2.0/auth/oss/token-create.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+INFLUX_ORG_ID=48c88459ee424a04
```
Comment (pierwill) [inline shared/text/api/v2.0/auth/oss/token-create.sh:1]: Let's make sure this isn't a real org ID.

File: shared/text/api/v2.0/auth/oss/token-create.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+INFLUX_API_TOKEN=YCiJXZPXSL
+
+curl -v --request POST \
+  http://localhost:8086/api/v2/authorizations \
+  --header "Authorization: Token ${INFLUX_API_TOKEN}" \
+  --header 'Content-type: application/json' \
+  --data '{
+  "status": "active",
+  "description": "IOT Center device 1",
```
Comment (pierwill) [inline shared/text/api/v2.0/auth/oss/token-create.sh:10]: Maybe a more generic name for the token would be better?

**Review by pierwill — APPROVED** [review]
Great stuff!

File: shared/text/api/v2.0/auth/oss/token-create.sh (hunk 3 above)
Comment (jstirnaman) [reply shared/text/api/v2.0/auth/oss/token-create.sh:1]: local

File: content/influxdb/v2.0/api-guide/api_intro.md (hunk 1 above)
Comment (jstirnaman) [reply content/influxdb/v2.0/api-guide/api_intro.md]: Thanks - debated removing it.

**Pull request merged**